echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/monitor/TriggerMonitor.java

```diff
@@ -90,7 +90,7 @@ protected void onEchoResponse(final TriggerEvent event) {
   protected Func1<Pipeline, Optional<Pipeline>> withMatchingTrigger(final TriggerEvent event) {
     val triggerPredicate = matchTriggerFor(event);
     return pipeline -> {
-      if (pipeline.getTriggers() == null) {
+      if (pipeline.getTriggers() == null || pipeline.isDisabled()) {
```

Comment:
Seems reasonable but ... should we also (or maybe instead) do this check in `orca`. 

Doing the check in `orca` would also prevent anyone from programmatically kicking off a disabled pipeline.

Not sure if `orca` would throw an exception, if so we should probably also do the check here to eliminate noise.